### ISSUE TITLE
[1.0.0] Metrics HTTP server causes websocket server to only accept local connections

### ISSUE THREAD

**Issue by tomjohn1028** [anycable/anycable-go#122]
## Tell us about your environment

**AnyCable-Go version: 1.0.0-33085f2**

**AnyCable gem version: anycable (1.0.1)**

### What did you do?
Ran the following on an AWS EC2 instance 
`anycable-go --host=ANYCABLE_GO_HOST --port=3334 --headers=cookie,origin --metrics_http=/metrics`

### What did you expect to happen?
I expected an anycable-go server to be listening for connections and _accepting outside connections_

### What actually happened?
The anycable-go server started up, but would not accepted outside connections.

`lsof -i tcp:3334` yields the following: 
````
COMMAND    PID USER   FD   TYPE  DEVICE SIZE/OFF NODE NAME
anycable- 9052 root    7u  IPv4 9482843      0t0  TCP localhost:directv-web (LISTEN)
````

The error seems to be coming from the metrics server. When I remove the _metrics_http_ flag 
(i.e. `anycable-go --host=ANYCABLE_GO_HOST --port=3334 --headers=cookie,origin`)

`lsof -i tcp:3334` yields the following: 
````
COMMAND    PID USER   FD   TYPE  DEVICE SIZE/OFF NODE NAME
anycable- 14309 root    7u  IPv4 9491652      0t0  TCP ip-172-31-17-110.us-west-2.compute.internal:directv-web->ip-172-31-10-21.us-west-2.compute.internal:35124 (ESTABLISHED)
anycable- 14309 root    8u  IPv4 9491099      0t0  TCP ip-172-31-17-110.us-west-2.compute.internal:directv-web (LISTEN)
anycable- 14309 root    9u  IPv4 9492487      0t0  TCP ip-172-31-17-110.us-west-2.compute.internal:directv-web->ip-172-31-10-21.us-west-2.compute.internal:35138 (ESTABLISHED)
anycable- 14309 root   12u  IPv4 9491444      0t0  TCP ip-172-31-17-110.us-west-2.compute.internal:directv-web->ip-172-31-10-21.us-west-2.compute.internal:35122 (ESTABLISHED)
anycable- 14309 root   13u  IPv4 9492485      0t0  TCP ip-172-31-17-110.us-west-2.compute.internal:directv-web->ip-172-31-10-21.us-west-2.compute.internal:35130 (ESTABLISHED)
anycable- 14309 root   14u  IPv4 9491453      0t0  TCP ip-172-31-17-110.us-west-2.compute.internal:directv-web->ip-172-31-10-21.us-west-2.compute.internal:35126 (ESTABLISHED)
...
````

**Comment by palkan** [comment]
Looks like related to https://github.com/anycable/anycable-go/issues/107 and was fixed in #108. Could you please try to upgrade to 1.0.1?

**Comment by tomjohn1028** [comment]
@palkan 1.0.1 seems to have the issues resolved. Thank you!
